### ISSUE TITLE
Devices show all entities but no stream

### ISSUE THREAD

**Issue by timknowlden** [fwestenberg/reolink_dev#354]
RLN8-410 NVR integrated with 2 x POE and 1 x E1 Zoom.

E1 Zoom all works
2 x POE camera show all entities and thumbnail but no stream, tried different resolutions, h264 & 265. RTMP and RTSP. Rebooting both cameras, NVR, Home assistant.

Home Assistant 2021.10.6
Latest Reolink_Dev

**Comment by timknowlden** [comment]
Home A
<img width="2332" alt="Screenshot 2021-11-06 at 14 46 02" src="https://user-images.githubusercontent.com/73787006/140613709-ea7944a8-9e95-4dea-b7b4-9a4c9c577309.png">
s

**Comment by timknowlden** [comment]
<img width="2332" alt="Screenshot 2021-11-06 at 14 47 16" src="https://user-images.githubusercontent.com/73787006/140613738-e5464a80-afa7-4c58-95e9-7ace45dec5fa.png">

**Comment by cpainchaud** [comment]
@timknowlden  Can you please try v0.37 ?

**Comment by timknowlden** [comment]
> @timknowlden Can you please try v0.37 ?

Hi, This does not allow playback of main stream in h264 or h265, rtsp or rtmp still.

**Comment by Rubn86** [comment]
Same issue on RLC-822A

**Comment by janneHalttu** [comment]
Same problem here. 4pcs RLC-810 with NVR

**Comment by timknowlden** [comment]
> Same problem here. 4pcs RLC-810 with NVR

Set stream to ext. motion sensors don’t seem to work but the stream does 
![9EC9DAF5-D63A-4AE9-A60C-302D10EC9FBD](https://user-images.githubusercontent.com/73787006/146668879-560bc7c5-a6c5-403e-9e9f-7440c59e98bd.png)

**Comment by janneHalttu** [comment]
That did not help me... =(

I found newer FW for NVR 3.0.0 and after that... I cannot connect it it anymore at all.. It  just says "something goes wrong" when i remome integration and try to install is again.. SO now I dont have cameras in HA anymore..

**Comment by timknowlden** [comment]
Oh.. I’m running 3.0.0 NVR firmware. Can’t be of any more help really. Hopefully someone else can solve. 
![Uploading 051E82F7-E81C-4E64-B553-5350750A4832.jpeg…]()

**Comment by janneHalttu** [comment]
I got new firmware and now camera is found and all 6 sensors. But still cannot see stream. 
Do I have to restart between changing h264/265 Main/sub/ext rtsp/rtmp?

**Comment by timknowlden** [comment]
> I got new firmware and now camera is found and all 6 sensors. But still cannot see stream. Do I have to restart between changing h264/265 Main/sub/ext rtsp/rtmp?
> 
> I can see snapshop on all cameras, but stream does not work..

Shouldn’t have to restart. Just maybe reload integration?

do your motion sensors work?

**Comment by janneHalttu** [comment]
Yes.. My sensors work. Only issue seems to be stream not working..
I have tried ALL combinations, Recommended ext/264 and default h265/sub or main..

**Comment by janneHalttu** [comment]
I got this help from customer support:
RTSP URL is usually used on third party video player, such as VLC player. It cannot be used directly on web browser, since web browsers don't support RTSP protocol. (or you may install RTSP plug-in on web browser to view RTSP stream.)

The RTSP URL can be used both in LAN and WAN. If you want to view RTSP stream in WAN, you will need to do port forwarding on the router.

The format of RTSP address is as below:
rtsp://(user name):(password)@(IP address):554/h264Preview_01_(stream type)

For example, if your camera or NVR has the following information,
IP address: 192.168.10.92
username: admin
password: 111111

then the RTSP URL will be:

Main Stream: rtsp://admin:111111@192.168.10.92:554/h264Preview_01_main

Sub Stream: rtsp://admin:111111@192.168.10.92:554/h264Preview_01_sub
 

Note: For Reolink POE NVRs (with 4MP/5MP cameras connected), the username and password in the URL should be the ones for NVR. You may also change the channel number to view different channels.

For example, if you want to view channel 2 on NVR, then the RTSP URL will be:

Main Stream: rtsp://admin:111111@192.168.10.92:554/h264Preview_02_main

Sub Stream: rtsp://admin:111111@192.168.10.92:554/h264Preview_02_sub

**Comment by janneHalttu** [comment]
I just tested and above settings work

**Comment by janneHalttu** [comment]
Sub Stream: rtsp://admin:111111@192.168.10.92:554/h264Preview_02_sub
This works in WebRTC 
And on Integration settings rtsp/H265/sub works... BUT I just noticed after testing that YOU HAVE TO RESTART HA TO MAKE SETTINGS CHANGE APPLY!
